best-practices/README.md

```diff
@@ -187,6 +187,13 @@ Email
 [MailView]: https://github.com/37signals/mail_view
 [ActionMailer Preview]: http://api.rubyonrails.org/v4.1.0/classes/ActionMailer/Base.html#class-ActionMailer::Base-label-Previewing+emails
 
+Web
+---
+
+* Avoid a Flash of Unstyled Text, even when no cache is available.
+* Avoid rendering delays caused by synchronous loading.
+* Avoid linking to assets over http.
```

Comment:
It means we should use `https` instead. When using external font services, trackers, and widgets, I've seen a few cases where people bleed information by using `http`.